compiler+runtime/src/cpp/jank/runtime/obj/ratio.cpp

```diff
@@ -0,0 +1,556 @@
+#include <jank/runtime/obj/ratio.hpp>
+#include <fmt/compile.h>
+#include <limits>
+
+namespace jank::runtime
+{
+
+  constexpr native_real EPSILON = std::numeric_limits<native_real>::epsilon();
```

Comment:
Please use direct initialization and `auto` here. Also, in jank's code we don't use SCREAMINGCASE for anything.

```cpp
constexpr static auto epsilon{ std::numeric_limits<native_real>::epsilon() };
```

Reply:
updated.